<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImpBFNETWiz</class>
 <widget class="QDialog" name="ImpBFNETWiz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Biblioteka Filmów - Kreator importu z BF NET</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>4</y>
     <width>317</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Podaj ścieżkę do pliku csv pochodzacego z Biblioteki Filmów NET</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_csv_BFNET">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>20</y>
     <width>533</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Sel_BFNET">
   <property name="geometry">
    <rect>
     <x>552</x>
     <y>16</y>
     <width>49</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>48</y>
     <width>529</width>
     <height>85</height>
    </rect>
   </property>
   <property name="title">
    <string>Wybierz co zaimportować</string>
   </property>
   <widget class="QCheckBox" name="checkBox_Obsada">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>20</y>
      <width>105</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Tabela Obsada</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Wyp">
    <property name="geometry">
     <rect>
      <x>136</x>
      <y>20</y>
      <width>125</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Tabele Wypozyczeń</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_LZ">
    <property name="geometry">
     <rect>
      <x>272</x>
      <y>20</y>
      <width>165</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Tabela Lokalizacje zdjęciowe</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Ocena">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>44</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Tabela Ocena</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Prod">
    <property name="geometry">
     <rect>
      <x>136</x>
      <y>44</y>
      <width>117</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Tabela Producent</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Dyst">
    <property name="geometry">
     <rect>
      <x>272</x>
      <y>44</y>
      <width>125</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Tabela Dystrybutor</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_IMP">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>144</y>
     <width>93</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Zaimportuj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_CANCEL">
   <property name="geometry">
    <rect>
     <x>456</x>
     <y>144</y>
     <width>75</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Anuluj</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_IMP</sender>
   <signal>clicked()</signal>
   <receiver>ImpBFNETWiz</receiver>
   <slot>BTN_IMP_CLICKED()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>391</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_CANCEL</sender>
   <signal>clicked()</signal>
   <receiver>ImpBFNETWiz</receiver>
   <slot>BTN_CANCEL_CLICKED()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>502</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>569</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Sel_BFNET</sender>
   <signal>clicked()</signal>
   <receiver>ImpBFNETWiz</receiver>
   <slot>BTN_SEL_PATH_CLICKED()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>584</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>593</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>BTN_IMP_CLICKED()</slot>
  <slot>BTN_CANCEL_CLICKED()</slot>
  <slot>BTN_SEL_PATH_CLICKED()</slot>
 </slots>
</ui>
